test/type.coffee

```diff
@@ -8,19 +8,33 @@ Amen.describe "Type functions", (context) ->
     isString, isFunction, isObject, isArray, isRegExp, isDate, isDefined,
     isGenerator, isPromise} = require "../src/type"
 
-  context.test "type"
-
-  context.test "isType"
-
-  context.test "instanceOf"
+  context.test "type", ->
+    assert type 7
+    assert type "7"
+    assert type true
+
+  context.test "isType", ->
+    assert isType(String, "7")
+    assert isType(Number, 7)
+    assert isType(Boolean, true)
+    assert ! isType(Number, false)
+
+  context.test "instanceOf", ->
```

Comment:
@CodingBaer thank you again for the patch. I'd love to see the `assert` for the type actually check the result, ex: 

```coffee
assert Number == type 7
```